templates/includes/sidebar.html

```diff
@@ -0,0 +1,6 @@
+<ul class="sidebar-nav">
+  <li><a href="{% url 'taxi:index' %}">Home page</a></li>
+  <li><a href=#>Manufacturers</a></li>
+  <li><a href=#>Cars</a></li>
+  <li><a href=#>Drivers</a></li>

```

Comment:
The links for 'Manufacturers', 'Cars', and 'Drivers' are placeholders (`href=#`). Once the corresponding views and URL patterns are set up, update these links to use the `{% url %}` template tag to point to the correct paths.